README.md

```diff
@@ -12,23 +12,26 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
   * Increased inclusion,
   * Increased ease of use
 * Who is it for?
-  * US based astronomers??  is this true??  all astronomers??
+  * All astronomers, particularly those interested in analysis of NASA astrophysics data. 
 * Limits: What does it not do?
   * How many cores/RAM do I get?
+    * There are several options for the size of the compute.  Please select the smallest that you can use for testing and exploration.  Do not use the larger images unless you have already tested a smaller subset of the analysis on a smaller compute instance.    
     * In JK's understanding (which needs to be updated) you get what the numbers say when you choose a server upon login, but for a limited time if you need slightly more, the code will not crash but will have access to slightly more. 
     * Raen has seen some evidence of this "bursting" behavior, but doesn't have direct knowledge of the actual configuration. It would be good to know.
   * How much disk space do I have access to?
     * Current default is 10GB (Feb 2024).
+    * This can be increased on request.  
   * What restrictions are there to prevent egress charges?
     * Raen's understanding is:
```

Comment:
Now that we have a succinct answer from Tess, I think we can remove the historical aside and some other info that might be too much for users. 
```suggestion
   
```